### PR TITLE
removed all framework cruft, simplified where necessary

### PR DESCRIPTION
**Pull request by jpickell**
There's really no need for the Bootstrap framework as it unnecessarily complicates the code and hides portions that as a developer, you should become familiar with.  (especially the formatting issues surrounding the transforms) 

I removed all of the framework specific code, and re-wrote the CSS to provide a similar desktop appearance and created a simple and legible mobile design.

The css/html is very simple and straightforward now and you should be able to tweak it to fit whatever design you desire.

**tldr:  Don't use frameworks unless absolutely necessary.**   😄

### CODE REVIEW

**Comment by jpickell** [comment]
Here's a couple of screenshots...

Desktop:
egpaul-desktop

Mobile:
egpaul-mobile

**Comment by egpaul** [comment]
That looks awesome! Thank you for taking the time and updating it. I kept trying to work around using bootstrap I guess when I shouldn’t have.
